doc/cli_vs_yum.rst

```diff
@@ -211,6 +211,8 @@ There is no shell in DNF but the case above is still valid. We provide the ``--a
 
 This command is equal to ``yum swap A B``.
 
+DNF provides swap command but only ``dnf swap A B`` syntax is supported
```

Comment:
What does this mean? Was there more to the `yum swap` syntax that isn't supported?